src/PileupTrack.js

```diff
@@ -222,7 +273,8 @@ class NonEmptyPileupTrack extends React.Component {
 
     var referenceSource = this.props.referenceSource;
     var vReads = this.state.reads.map(
-        read => this.addRead(read, referenceSource));
+            read => this.addRead(read, referenceSource))
+        .filter(read => read.refLength);  // drop alignments w/o CIGARs
```

Comment:
It's actually fairly common (see #111). I'm not sure what alignments without Cigar strings correspond to—maybe @arahuja or @smondet know more.